extensions/cdc-postgres/src/main/java/com/hazelcast/jet/cdc/postgres/PostgresCdcSources.java

```diff
@@ -0,0 +1,325 @@
+                .exclusive("table.whitelist", "table.blacklist");
+
+        private final DebeziumConfig config;
+
+        /**
+         * @param name name of the source, needs to be unique,
+         *             will be passed to the underlying Kafka
+         *             Connect source
+         */
+        private Builder(String name) {
+            Objects.requireNonNull(name, "name");
+
+            config = new DebeziumConfig(name, "io.debezium.connector.postgresql.PostgresConnector");
+            config.setProperty(CdcSource.SEQUENCE_EXTRACTOR_CLASS_PROPERTY, PostgresSequenceExtractor.class.getName());
+            config.setProperty(CdcSource.DB_SPECIFIC_EXTRA_FIELDS_PROPERTY, "schema");
+        }
+
+        /**
+         * IP address or hostname of the database server, has to be
+         * specified.
+         */
+        @Nonnull
+        public Builder setDatabaseAddress(@Nonnull String address) {
+            config.setProperty("database.hostname", address);
+            return this;
+        }
+
+        /**
+         * Optional port number of the database server, if unspecified
+         * defaults to the database specific default port (5432).
+         */
+        @Nonnull
+        public Builder setDatabasePort(int port) {
+            config.setProperty("database.port", Integer.toString(port));
+            return this;
+        }
+
+        /**
+         * Database user for connecting to the database server. Has to
+         * be specified.
+         */
+        @Nonnull
+        public Builder setDatabaseUser(@Nonnull String user) {
+            config.setProperty("database.user", user);
+            return this;
+        }
+
+        /**
+         * Database user password for connecting to the database server.
+         * Has to be specified.
+         */
+        @Nonnull
+        public Builder setDatabasePassword(@Nonnull String password) {
+            config.setProperty("database.password", password);
+            return this;
+        }
+
+        /**
+         * The name of the PostgreSQL database from which to stream the
+         * changes. Has to be set.
+         * <p>
+         * Currently this source is not capable of monitoring multiple
+         * databases, only multiple schemas and/or tables. See white-
+         * and black-listing configuration options for those.
+         */
+        public Builder setDatabaseName(String dbName) {
+            config.setProperty("database.dbname", dbName);
+            return this;
+        }
+
+        /**
+         * Logical name that identifies and provides a namespace for the
+         * particular database server/cluster being monitored. The
+         * logical name should be unique across all other connectors.
+         * Only alphanumeric characters and underscores should be used.
+         * Has to be specified.
+         */
+        @Nonnull
+        public Builder setClusterName(@Nonnull String cluster) {
```

Comment:
I wonder if this is relevant to us, since in Debezium it only affect the Kafka topic name? Can we just use a random name?

Reply:
Yes, seems like we can.